src/pocketmine/tile/Sign.php

```diff
@@ -64,6 +64,27 @@ public function setText($line1 = "", $line2 = "", $line3 = "", $line4 = ""){
 
 		return true;
 	}
+	
+	/**
+	 * @param int    $index 0-3
+	 * @param string $line
+	 * @param bool $update
```

Comment:
@SOF3 It's funny how hard it is to actually get such a simple commit right... i do it too